ddtrace/tracer/remote_config_test.go

```diff
@@ -41,7 +41,7 @@ func TestOnRemoteConfigUpdate(t *testing.T) {
 
 		// Telemetry
 		telemetryClient.AssertNumberOfCalls(t, "ConfigChange", 1)
-		telemetryClient.AssertCalled(t, "ConfigChange", []telemetry.Configuration{{Name: "trace_sample_rate", Value: 0.5, Origin: "remote_config"}})
+		telemetryClient.AssertCalled(t, "ConfigChange", []telemetry.Configuration{{Name: "trace_sample_rate", Value: 0.5, Origin: originRemoteConfig}})
```

Comment:
just to check my understanding, when a value is updated through remote config the telemetry event will be set with the remote_config origin?

Reply:
@avivenzio-dd Yes, it will be set with `remote_config` origin.